lib/install/validate/network.go

```diff
@@ -455,6 +455,12 @@ func (v *Validator) getNetwork(op trace.Operation, name string) (object.NetworkR
 		return nil, err
 	}
 	if len(nets) > 1 {
+		for _, net := range nets {
```

Comment:
Could we just tell the users to use network inventory path instead of picking one from the array?

Reply:
Good advice, i will modify it.